.github/workflows/render_rmarkdown.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    paths: ['**.Rmd']
+
+name: render-rmarkdown
+
+jobs:
+  render-rmarkdown:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+      
+      - name: Render Rmarkdown files and Commit Results
+        run: |
+          Rscript -e "rmarkdown::render_site()"
+          if [[ "$(git status --porcelain)" != "" ]]; then
+            git config --global user.name "$GITHUB_ACTOR"
+            git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
Who will then appear as the author of the commit when we use that "actor" variable? Maybe we can also just set it to something friendly-looking, like "Dashboard Updater" or something like that? That said, I'm not sure the email is necessary - have you tried running that action?

Reply:
I agree that email is not necessary. When I run the action, the github actor var  just indicated me as a user. It might be more readable to keep.